storybook/web/webpack.config.js

```diff
@@ -1,5 +1,27 @@
 /* eslint-disable no-param-reassign */
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
 module.exports = ({ config }) => {
   config.resolve.extensions = ['.web.js', '.mjs', '.js', '.jsx', '.json'];
+  config.module.rules.push({
+    test: /\.(woff|woff2|eot|ttf|otf)$/,
+    use: [
+      {
+        loader: 'file-loader',
+        query: {
+          name: 'fonts/[name].[ext]',
+        },
+      },
+    ],
+  });
+  config.plugins.push(
+    new CopyWebpackPlugin([
+      {
+        from: 'public/fonts',
+        to: 'fonts',
```

Comment:
Why do we need this? Can't we directly do `url('public/fonts/Lato-Regular.ttf')`

Reply:
Because the folder that’s published is `docs` hence it won’t resolve the url. Or I have to publish public as well which is incorrect. And that’s were copywebpack plugin comes in and meant for such use cases.